docs/sidebars.js

```diff
@@ -105,7 +105,17 @@ const sidebars = {
             id: "concepts/foundation/communication/main",
           },
           items: [
-            "concepts/foundation/communication/public_private_calls",
+            {
+              label: "Public <> Private Communication",
+              type: "category",
+              link: {
+                type: "doc",
+                id: "concepts/foundation/communication/public_private_calls/main",
+              },
+              items: [
+                "concepts/foundation/communication/public_private_calls/slow_updates_tree"
```

Comment:
will this show as "slow updates tree"? Because a new reader would have no idea what it means. Better to say "read historical public data in private "

Reply:
yeah it says read historical public data in private in the concepts (and slow updates in the syntax), it'll say slow-updates-tree in the URL tho